test/recipes/30-test_evp_data/evpmac_common.txt

```diff
@@ -400,3 +400,12 @@ Custom = "My Tagged Application"
 Output = D5BE731C954ED7732846BB59DBE3A8E30F83E77A4BFF4459F2F1C2B4ECEBB8CE67BA01C62E8AB8578D2D499BD1BB276768781190020A306A97DE281DCC30305D
 Ctrl = size:64
 Ctrl = xof:1
+
+Title = KMAC long customisation string
```

Comment:
Do you need to quote the source of the test vector?

Reply:
No, I created it and I'd like it replaced by a vector from NIST.

Reply:
I've added a vector from the ACVP and noted the source in the test data.